src/components/common/Toast/Toast.style.ts

```diff
@@ -0,0 +1,48 @@
+import { css, keyframes, Theme } from '@emotion/react';
+import { flexGenerator } from '@styles/generator';
+
+export const toastWrapperStyle = (theme: Theme, isVisible: boolean, toastBottom: number) => css`
```

Comment:
```suggestion
export const toastWrapperStyle = (isVisible: boolean, toastBottom: number) => (theme: Theme) => css`
```

이렇게하면 컴포넌트 쪽 파일에서 theme import 받고 인자로 다시 넘겨주고 안해도 돼요!

Reply:
오와 이렇게 되는지 몰랐어요 감사합니다 ㅎㅎ